PubnativeLite/PubnativeLite/Ad Request/PNLiteAdFactory.m

```diff
@@ -46,13 +46,22 @@ - (PNLiteAdRequestModel *)createAdRequestWithZoneID:(NSString *)zoneID
     
     NSString* privacyString =  [[HyBidUserDataManager sharedInstance] getIABUSPrivacyString];
     if (!([privacyString length] == 0)) {
-        adRequestModel.requestParameters[@"usprivacy"] = privacyString;
+        adRequestModel.requestParameters[HyBidRequestParameter.usprivacy] = privacyString;
     }
     
-    if (![HyBidSettings sharedInstance].coppa && ![[HyBidUserDataManager sharedInstance] isCCPAOptOut]) {
+    if (![HyBidSettings sharedInstance].coppa && [[HyBidUserDataManager sharedInstance] canCollectData] && ![[HyBidUserDataManager sharedInstance] isCCPAOptOut]) {
```

Comment:
I'm not sure we need to add this can collect data there since we are not removing params 'cause of GDPR non-consent

Reply:
Removed that check in the latest commit. Please check @eros902002  😘